src/plugins/core/java.rs

```diff
@@ -174,8 +174,21 @@ impl JavaPlugin {
             file::rename(entry.path(), dest)?;
         }
 
+        if os() == "macosx" {
```

Comment:
this is nitpicky and arguably personal preference but I'd prefer this
```suggestion
        cfg!(target_os = "macos") {
```

Reply:
I'm not sure but I think there are some (certainly meaningless in this case) performance benefits to this. In theory the compiler should be able to just remove the conditional entirely